tests/e2e/lib/blocks/eventbrite.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Internal dependencies
+ */
+import { waitForSelector, waitAndClick, waitAndType } from '../page-helper';
+
+export default class EventbriteBlock {
+	constructor( block, page ) {
+		this.blockTitle = EventbriteBlock.title();
+		this.block = block;
+		this.page = page;
+		this.blockSelector = '#block-' + block.clientId;
+	}
+
+	static name() {
+		return 'eventbrite';
+	}
+
+	static title() {
+		return 'Eventbrite';
+	}
+
+	static eventId() {
+		return '96820156695';
+	}
+
+	static embedUrl() {
```

Comment:
Do we really need to have this method to be static? I see that it used only once inside instance method.

Reply:
Good point, moved to instance method.